.drone.yml

```diff
@@ -1,130 +1,133 @@
 ---
 kind: pipeline
 name: amd64
+type: docker
 
 platform:
   os: linux
   arch: amd64
 
 steps:
-- name: build
```

Comment:
Most of the changes to this file (excepting the `type: docker` and the `integration-test` pipeline) seem to be formatting changes. Was that the intention? If so, it may be helpful in the future to separate out larger formatting changes like this into their own commit - when they get mixed in with real changes like this it can be hard to understand the changes.

Reply:
`Type: docker` was added because the old pipeline was working based on undefined drone behavior.
The [docs](https://docs.drone.io/pipeline/configuration/) don't state anywhere that `Type` can be set to nil and that docker will be used by default.

I am unsure if having a commit that just corrects our inconsistent file indentations is useful. Since GitHub can show the spacing change in the diff.

Reply:
> Type: docker was added because the old pipeline was working based on undefined drone behavior.
The [docs](https://docs.drone.io/pipeline/configuration/) don't state anywhere that Type can be set to nil and that docker will be used by default.

Ok - I assume that you are referring to the logic outlined [here](https://discourse.drone.io/t/type-parameter-is-not-required/9029). Seems like these may have been made in older versions of drone and needed an update. Thank you for doing that. 

> I am unsure if having a commit that just corrects our inconsistent file indentations is useful. Since GitHub can show the spacing change in the diff.

You don't need to make a change to accommodate me here. That being said, when reviewing it doesn't show line by line indentations very well. It shows massive blocks of read and green (e.x. the first block in this PR is 40 lines) - and I have to compare line by line to make sure that indentation was the only thing that changed. This is made especially important since you changed other things in this file, so I have no indicator of if anything in this section was a spacing/linting change or a functionality change. This takes more time, and makes it harder to review.

Reply:
ahh, I see maybe I can split for a review and then squash the commit after?

Reply:
Yeah - you could do that. I'm would also be ok with you just leaving the commit as separate. All good though, we don't need to do anything for this PR right now.